frontend/src/modules/machinery/components/Machinery.js

```diff
@@ -29,9 +28,13 @@ type Props = {|
  * third-party Machine Translation.
  */
 export default class Machinery extends React.Component<Props> {
+    submitForm = (event: SyntheticKeyboardEvent<>) => {
+        event.preventDefault();
+        this.props.searchMachinery(event.currentTarget[0].value);
```

Comment:
I'm not sure how to address the flow error here. Should I just ignore it?

Reply:
This might be simpler and more robust (especially if we ever change the DOM structure) by using a ref on the input.